src/main/python/pybuilder/plugins/python/pylint_plugin.py

```diff
@@ -44,4 +45,63 @@ def execute_pylint(project, logger):
     logger.info("Executing pylint on project sources")
 
     command_and_arguments = ["pylint"] + project.get_property("pylint_options")
-    execute_tool_on_modules(project, "pylint", command_and_arguments, True)
+    pylint_output_file_path = execute_tool_on_modules(project, "pylint", command_and_arguments, True)[1]
+
+    with open(pylint_output_file_path, 'r') as f:
+        file_content = f.read().splitlines()
+
+    module_name = ''
+    pylint_score = 0
+    pylint_change = 0
+    errors = 0
+    errors_info = ''
+    warnings = 0
+    warnings_info = ''
+    show_info_messages = project.get_property('pylint_show_info_messages')
+    show_warning_messages = project.get_property('pylint_show_warning_messages')
```

Comment:
I could set show_info_messages = True and show_info_messages = True when verbose and debug are true. What do you think about it @AlexeySanko ?

Reply:
Hi, @maciejpolanczyk 
Good idea. Thanks!